lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -79,8 +79,8 @@ module.exports = [
               {resourceType: 'font', requestCount: 2, size: '80000±1000'},
               {resourceType: 'script', requestCount: 3, size: '55000±1000'},
               {resourceType: 'image', requestCount: 2, size: '28000±1000'},
-              {resourceType: 'document', requestCount: 1, size: '2100±100'},
-              {resourceType: 'other', requestCount: 2, size: '1250±50'},
+              {resourceType: 'document', requestCount: 1, size: '2200±100'},
+              {resourceType: 'other', requestCount: 2, size: '1300±50'},
```

Comment:
really wish we knew what these two other requests were right about now...

Reply:
fonts 🦀 🦀 🦀

Reply:
but we breakout fonts into their own thing already?

Reply:
I'm going off memory here. I'll fire it up...

![image](https://user-images.githubusercontent.com/4071474/62803008-eb85c880-baae-11e9-828f-7e4a5c34760e.png)

ok yeah i was mistaken. fetch and the favicon

(sidenote - I want to add a favicon to the static server eventually, it keeps coming up as a "difference" when running smoke tests in different environments. Only in the CLI case are we attached to the protocol while a favicon is requested - in LR and DT we connect too late).

Reply:
I wonder if a "fetch" should dig a little deeper for categorizing?

Reply:
yeah that's certainly interesting, I would consider fetch worthy of an XHR/API category all its own anyhow but probably a topic worth bringing up in a different thread :)

want to throw the comment on there that it's the favicon and fetch for now?

Reply:
done